r/tests/testthat/test-data-type.R

```diff
@@ -395,6 +395,18 @@ test_that("decimal type and validation", {
   expect_error(decimal(4, NA), '"scale" must be an integer')
 
   expect_r6_class(decimal(4, 2), "Decimal128Type")
```

Comment:
Do we want to also do something like `expect_identical(class(decimal(4, 2)), class(decimal128(4, 2))` that asserts that `decimal()` -> `decimal128()`

Reply:
Sure. Done